### PR TITLE
fix(components): [drawer] The animation direction of left/top/bottom …

### PR DESCRIPTION
**Pull request by AcWrong02**
…of the drawer component is wrong

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #689 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
- 当前组件还没有添加任何测试
- pnpm dev后，就能看到对应组件的动画效果

### CODE REVIEW

File: pnpm-lock.yaml
Comment (zzcr) [inline pnpm-lock.yaml:1]: 你好，感谢您的贡献，这个pnpm-lock不需要提交哈

Comment (zzcr) [reply pnpm-lock.yaml:1]: @AcWrong02

**Review by Huangyilin19 — APPROVED** [review]

File: pnpm-lock.yaml
Comment (AcWrong02) [reply pnpm-lock.yaml:1]: > 你好，感谢您的贡献，这个pnpm-lock不需要提交哈

你好，我把pnpm-lock.yaml删掉了，这样子可以吗？

**Review by zzcr — APPROVED** [review]

**Comment by AcWrong02** [comment]
@all-contributors please add @AcWrong02  for fixing component bugs

**Pull request merged**